front/lib/actions/mcp_internal_actions/servers/agent_router.ts

```diff
@@ -20,12 +20,32 @@ const serverInfo: InternalMCPServerDefinitionType = {
   documentationUrl: null,
 };
 
+export const LIST_ALL_AGENTS_TOOL_NAME = "list_all_published_agents";
+export const SUGGEST_AGENTS_TOOL_NAME = "suggest_agents_for_content";
+
+const SERVER_INSTRUCTIONS = `This toolset provides access to specialized agents available in the workspace.
+A good usage of these tools is:
```

Comment:
I think even that falls in the "instructions" bucket and should not live on the server instructions